CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(aruco_catkin)
+
+find_package(catkin_simple REQUIRED)
+catkin_simple(ALL_DEPS_REQUIRED)
+
+include(ExternalProject)
+
+file(MAKE_DIRECTORY ${CATKIN_DEVEL_PREFIX}/include)
+
+ExternalProject_Add(aruco_src
+  URL https://sourceforge.net/projects/aruco/files/3.0.0/aruco304.zip
+
+  CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${CATKIN_DEVEL_PREFIX}
```

Comment:
Also add `-DCMAKE_BUILD_TYPE:STRING=Release`, should make a big difference in performance